src/components/graphql-provider/__mocks__/graphql-provider.tsx

```diff
@@ -382,11 +542,29 @@ export const GraphQlProvider = ({
   return (
     <MockedProvider
       mocks={[
```

Comment:
Is there a way around this?

Reply:
This looks like a temporary solution which will be properly fixed at some point soon.